commit fbe13fc840030e7f032cbf68c1f6eb672e35e97b
Author: Matthew Booth <mbooth@redhat.com>
Date:   Mon Sep 2 14:25:07 2013 +0100

    build: Use preferred Module::Build->do_system in favour of system()

diff --git a/Build.PL b/Build.PL
index cd32570..7414463 100644
--- a/Build.PL
+++ b/Build.PL
@@ -17,7 +17,7 @@ sub ACTION_po
 {
     my $self = shift;
 
-    system($self->config('make'), '-C', 'po', 'pot', 'update-po') == 0
+    $self->do_system($self->config('make'), '-C', 'po', 'pot', 'update-po')
         or return 1;
 
     return 0;
@@ -30,8 +30,8 @@ sub process_locale_files
 
     # Make them first
     $self->log_info("process_locale_files\n");
-    system($self->config('make'), '-C', 'po', 'install',
-           'LOCALEDIR=../blib/locale') == 0
+    $self->do_system($self->config('make'), '-C', 'po', 'install',
+                     'LOCALEDIR=../blib/locale')
         or die(); # We need to die() here because process_foo_files return isn't
                   # checked.
 }
@@ -148,12 +148,12 @@ sub ACTION_changelog
         return 1;
     }
 
-    unless(system("git", "submodule", "init") == 0) {
+    unless($self->do_system("git", "submodule", "init")) {
         $self->log_info("Failed to initialise git2cl submodule: $@\n");
         return 1;
     }
 
-    unless(system("git", "submodule", "update") == 0) {
+    unless($self->do_system("git", "submodule", "update")) {
         $self->log_info("Failed to update git2cl submodule: $@\n");
         return 1;
     }
